src/dmd/console.d

```diff
@@ -42,17 +42,39 @@ enum Color : int
     white         = bright | lightGray,
 }
 
+/* The ANSI escape codes are used.
+ * https://en.wikipedia.org/wiki/ANSI_escape_code
+ * Foreground colors: 30..37
+ * Background colors: 40..47
+ * Attributes:
+ *  0: reset all attributes
+ *  1: high intensity
+ *  2: low intensity
+ *  3: italic
+ *  4: single line underscore
+ *  5: slow blink
+ *  6: fast blink
+ *  7: reverse video
+ *  8: hidden
+ */
+
 struct Console
 {
     version (Windows)
     {
         import core.sys.windows.windows;
+        import core.sys.windows.wincon;
+        
+        // added in druntime#2285
+        static if (__VERSION__ < 2082)
+            enum ENABLE_VIRTUAL_TERMINAL_PROCESSING = 4;
 
       private:
         CONSOLE_SCREEN_BUFFER_INFO sbi;
         HANDLE handle;
         FILE* _fp;
-
+        bool vtmode;
+        bool resetVtmode;
```

Comment:
What do these names mean?

Reply:
`vtmode`: whether the console currently understands VT sequences.
`resetVtmode`: whether we need to turn off VT mode in the console when the compiler exits.

Reply:
These would certainly be helpful as comments in the code.